conf/messages

```diff
@@ -45,6 +45,7 @@ session_expired.heading = For your security, this service has been reset
 session_expired.guidance = The details you have given have been deleted because you did not continue the service for 1 hour.
 
 site.back = Back
+site.back.toStartPage = Back to selection page
```

Comment:
Is this a content change from the prototype?

Reply:
Yes. A bit weird if you ask me, but who am I to question?